tests/foreman/ui/test_gpgkey.py

```diff
@@ -111,16 +123,18 @@ def test_negative_create_2(self, name):
         @assert: gpg key is not created
 
         """
-
-        key_content = read_data_file(VALID_GPG_KEY_FILE)
+        kwargs = {
+            'key_content': self.key_content,
+            'name': name,
+            'org': self.organization.name,
+        }
         with Session(self.browser) as session:
-            make_gpgkey(session, org=self.org_name,
-                        name=name, key_content=key_content)
+            make_gpgkey(session, **kwargs)
             self.assertIsNotNone(self.gpgkey.search(name))
-            make_gpgkey(session, org=self.org_name,
-                        name=name, key_content=key_content)
-            self.assertTrue(self.gpgkey.wait_until_element
-                            (common_locators["alert.error"]))
+            make_gpgkey(session, **kwargs)
+            self.assertTrue(
```

Comment:
@Ichimonji10 sorry for the late reply but seems that you get all the assertions I meant. The thing is wait_until_element will return None if the element is not found and not a boolean. So is better to ensure that None is not returned. In practice this will not change because the python's feature to consider None as False. But this is intended to give more meaning when reading the code.